assets/scripts/util/formatting.js

```diff
@@ -0,0 +1,9 @@
+function numberWithCommas (x) {
```

Comment:
Let's use `[Intl.Numberformat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) for formatting numbers. That way we can guarantee we can format properly in other languages (not all of them use commas as digit separators, or periods as decimal separators).